src/cases/CmsBlockGroupTestCase.php

```diff
@@ -0,0 +1,103 @@
+<?php
+namespace luya\testsuite\cases;
+
+use yii\base\InvalidConfigException;
+
+/**
+ * Class CmsBlockGroupTestCase
+ *
+ * Testing a cms block group for the admin.
+ *
+ * Example usage:
+ * ```php
+ * namespace cmstests\src\frontend\blockgroups;
+ *
+ * use luya\testsuite\cases\CmsBlockGroupTestCase
+ *
+ * class ProjectGroupTest extends CmsBlockGroupTestCase
+ * {
+ *      $blockGroupClass = 'luya\cms\frontend\blockgroup\ProjectGroup';
+ *
+ *      $blockGroupIdentifier = 'project-group';
+ * }
+ * ```
+ *
+ * @package luya\testsuite\cases
+ * @author Alexander Schmid <alex.schmid@stud.unibas.ch>
+ * @since 1.0.11
+ */
+abstract class CmsBlockGroupTestCase extends WebApplicationTestCase
+{
+    /**
+     * @var string The path to the block group tested.
+     */
+    public $blockGroupClass;
+
+    /**
+     * @var \luya\cms\base\BlockGroup;
+     */
+    public $blockGroup;
+
+    /**
+     * @var string The used identifier
+     */
+    public $blockGroupIdentifier;
+
+    /**
+     * Provide configurtion array.
+     *
+     * This method can be overwritten if adjustments are needed.
+     * In most cases this will not be needed, therefore a default config is provided
+     */
+    public function getConfigArray()
+    {
+        return [
+            'id' => 'blockGroupTest',
+            'basePath' => dirname(__DIR__),
+        ];
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function afterSetup()
+    {
+        parent::afterSetup();
+
+        if (!$this->blockGroupClass) {
+            throw new InvalidConfigException("The 'blockGroupClass' property can not be empty.");
+        }
+        if (!$this->blockGroupIdentifier) {
+            throw new InvalidConfigException("The 'blockGroupIdentifier' property can not be empty.");
+        }
+
+        $class = $this->blockGroupClass;
+        $this->blockGroup = new $class();
+    }
+
+    /**
+     * Tests if the identifier is correct
+     */
+    public function testIdentifier()
```

Comment:
If someone wants to check if those test are working, and not having a risky test you could just return true.

So you could do $this->assertTrue($this->testIdentifier()); to make sure tests are not risky.

same for testLabel and testGetPosition